BookingsApi/BookingsApi.Contract/Requests/AddJusticeUserRequest.cs

```diff
@@ -44,7 +44,7 @@ public AddJusticeUserRequest()
     public string CreatedBy { get; set;}
     
     /// <summary>
-    /// The user's role. This can be a VHO or a Team Lead.
+    /// The user's roles. This can be a VHO or a Team Lead, and/or Staff Member.
     /// </summary>
-    public JusticeUserRole Role { get; set;}
+    public JusticeUserRole[] Role { get; set;}
```

Comment:
Could you change this to a List please. I believe other classes in the project use lists too

Reply:
changed